README.md

```diff
@@ -1,30 +1,76 @@
-# NZSL
```

Comment:
Should the base README have been changed? Are the changes int the document in progress file above supposed to be for a new readme or are there some changes in this file that need to be reviewed as well?